wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/urlfactory/DefaultFactoryUrl.java

```diff
@@ -53,8 +46,8 @@ public DefaultFactoryUrl withFactoryFileLocation(String factoryFileLocation) {
   }
 
   @Override
-  public String devfileFileLocation() {
-    return devfileFileLocation;
+  public Map<String, String> devfileFileLocations() {
+    return singletonMap(null, devfileFileLocation);
```

Comment:
can you please explain this `null`? It looks very strange to put value under null key in map and I'm afraid it would bite us later.

Reply:
Added comment in code